node/chainSimulator/components/cryptoComponents.go

```diff
@@ -105,6 +107,12 @@ func CreateCryptoComponents(args ArgsCryptoComponentsHolder) (factory.CryptoComp
 	instance.managedPeersHolder = managedCryptoComponents.ManagedPeersHolder()
 	instance.keysHandler = managedCryptoComponents.KeysHandler()
 
+	if args.BypassTxSignatureCheck {
+		instance.txSingleSigner = &cryptoMocks.SingleSignerStub{}
```

Comment:
please create a disabled component instead of using this uncompleted test stub

Reply:
done